packages/ensnode-utils/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "bundler",
+    "module": "ESNext",
+    "noEmit": true,
+    "lib": ["ES2022"],
+    "target": "ES2022",
+    "declaration": true,
+    "outDir": "dist",
+    "strict": true,
+    "allowImportingTsExtensions": true,
+    "emitDeclarationOnly": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true
+  },
+  "include": [
+    "src/**/*",
+    "../../apps/ensnode/src/lib/plugin-helpers.ts",
```

Comment:
this seems like an anti-pattern, what is it accomplishing?

Reply:
Yup, this wa not meant to land in there.